src/components/filter-btn-container/index.jsx

```diff
@@ -19,11 +20,22 @@ function FilterBtnContainer() {
 
   return (
     <div>
```

Comment:
As this div isn't holding any classname, please use Fragment here. Also please change all the divs in the project that doesn't contain any attribute.